notation.go

```diff
@@ -85,12 +85,12 @@ func Sign(ctx context.Context, signer Signer, repo registry.Repository, opts Sig
 		logger.Infof("Resolved artifact tag `%s` to digest `%s` before signing", ref.Reference, targetDesc.Digest.String())
 	}
 
-	sig, signerInfo, err := signer.Sign(ctx, targetDesc, opts)
+	sig, ants, signerInfo, err := signer.Sign(ctx, targetDesc, opts)
```

Comment:
Looks like `ants` is non-nil only when signer is pluginSigner with generate-envelope capability. How about renaming it to something like `pluginAnnotations`?